src/ppx/scene/scene_scene.cpp

```diff
@@ -122,5 +133,50 @@ ppx::Result Scene::AddNode(scene::NodeRef&& node)
     return ppx::SUCCESS;
 }
 
+scene::ResourceArrayIndexMap<scene::Sampler> Scene::GetSamplersArrayIndexMap() const
+{
+    auto array = mResourceManager->GetSamplers();
+
+    std::unordered_map<const scene::Sampler*, uint32_t> indexMap;
+    //
```

Comment:
Missing comment?

Reply:
No, this is just to side step clang-format aligning the variable names so far over to the right.